### PR TITLE
PentestCyprus 2.0 | Added 2 local write-ups & 1 external write-up link

### PR DESCRIPTION
**Pull request by stefanoschrs**
Added the web/teaser local write-up
Added the forensics/teaser local write-up
Added the reverse/grovyle-75 external writeup link

### CODE REVIEW

**Comment by tjgerot** [comment]
LGTM. @stefanoschrs Would you happen to know how many points that challenge was worth?

**Comment by stefanoschrs** [comment]
@tjgerot the 4 teaser challenges were uploaded just to prepare the participants for the CTF so they didn't have actually any points

**Comment by tjgerot** [comment]
I see. I'm not sure if we want to leave it as you did or to label the points as 0. Hopefully we can get @abpolym or @dhanvi in on this, just so we do it right the first time.

**Comment by stefanoschrs** [comment]
Or maybe something like `N/A` since it's not actually 0 points since it wasn't in the actual competition score count.

Get Outlook for Android<https://aka.ms/ghei36>



On Wed, Sep 28, 2016 at 6:17 AM +0300, "Thomas Gerot" <notifications@github.com<mailto:notifications@github.com>> wrote:


I see. I'm not sure if we want to leave it as you did or to label the points as 0. Hopefully we can get @abpolym<https://github.com/abpolym> or @dhanvi<https://github.com/dhanvi> in on this, just so we do it right the first time.

-

**Comment by stefanoschrs** [comment]
@abpolym @dhanvi @tjgerot Any updates on this?

**Comment by tjgerot** [comment]
@stefanoschrs If [this](https://gitter.im/ctfs/write-ups) doesn't come out to something within the next 12 hours, we'll move ahead.

**Comment by abpolym** [comment]
@tjgerot @stefanoschrs Hey guys sorry for the late response - So I would just remove the points appendix - so `challenge-name` instead of `challenge-name-0`.

**Comment by stefanoschrs** [comment]
Great!

File: pentest-cyprus-2/reverse/grovyle-75/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 **Category:** Reverse
 **Points:** 75
 **Solves:** 75
```
Comment (tjgerot) [inline pentest-cyprus-2/reverse/grovyle-75/README.md]: Same amount of solves as points?

File: pentest-cyprus-2/reverse/teaser/README.md [hunk 2]
```diff
@@ -1,14 +1,28 @@
 # PentestCyprus 2.0 | Reverse Teaser
 
 **Category:** Reverse
-**Points:** -
-**Solves:** -
+**Points:** N/A
+**Solves:** N/A
```
Comment (tjgerot) [inline pentest-cyprus-2/reverse/teaser/README.md]: If you don't know the solves, simply leave it blank.

File: pentest-cyprus-2/reverse/grovyle-75/README.md (hunk 1 above)
Comment (stefanoschrs) [inline pentest-cyprus-2/reverse/grovyle-75/README.md]: Yeap

File: pentest-cyprus-2/reverse/teaser/README.md (hunk 2 above)
Comment (stefanoschrs) [inline pentest-cyprus-2/reverse/teaser/README.md]: Changed it to N/A

**Comment by tjgerot** [comment]
@stefanoschrs Thanks for taking the time to make these changes!

**Pull request merged**